### PR TITLE
ubuntu1804 provisioning

### PR DESCRIPTION
**Pull request by davehouse**
* I used expect version 5.45.4
The releng jumphosts have expect version 5.44.1.15 and I've used that just as well.
* you'll need the relops ssh key set up. I have this in my .ssh/config:
```
Host moon-chassis-??.inband.releng.mdc?.mozilla.com moon-chassis-?.inband.releng.mdc?.mozilla.com
    User relops
    IdentityFile ~/.ssh/id_rsa_relops_2020-05-07
    KexAlgorithms +diffie-hellman-group14-sha1
    Ciphers +aes128-cbc
```

Example run:
```
./reimage_1804.sh 7 {1..3}  # to install ubuntu 1804 onto chassis 7 nodes 1-3
# it then executes like:
./reimage_1804.exp --chassis relops@moon-chassis-7.inband.releng.mdc1.mozilla.com --node C1N1 --boot-params ""
...
```

### CODE REVIEW

File: moonshot/reimage_watch.exp [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+send -- "\[D"
+expect -exact "\[08;12H"
+send -- " "
+sleep 1
+send -- "PUPPET_PASS="
+sleep 4
+send -- "$kickstart"
+sleep 5
+send -- ""
+sleep 60
```
Comment (davehouse) [inline moonshot/reimage_watch.exp:149]: The only real difference between this and the reimage.exp is this wait to "watch" the install start.

File: moonshot/reimage.exp [hunk 2]
```diff
@@ -45,13 +45,15 @@ if {$force_conservative} {
 	}
 }
 
-set timeout 60
-spawn ssh -oStrictHostKeyChecking=no "$ilo"
+set timeout 120
+spawn ssh -oKexAlgorithms=+diffie-hellman-group1-sha1 -oCiphers=+aes128-cbc -oStrictHostKeyChecking=no "$ilo"
```
Comment (davehouse) [inline moonshot/reimage.exp:49]: the hp ilo doesn't allow newer ciphers/algorithms common in recent openssh

File: moonshot/reimage.exp [hunk 3]
```diff
@@ -45,13 +45,15 @@ if {$force_conservative} {
...
 match_max 100000
+sleep 5
 expect "password: "
 send -- "${password}\r"
```
Comment (davehouse) [inline moonshot/reimage.exp:53]: if using the relops user+sshkey, we can remove this password arg and passing.

**Review by aerickson — APPROVED** [review]
looks sweet. thanks :)

**Comment by aerickson** [comment]
Fix pushed.

**Pull request merged**